### DISCUSSION TITLE
YT channel?

### DISCUSSION THREAD

**Discussion by Anan7Codes** [BackendWorks/nestjs-microservices#8] [Q&A]
Do you have a YT channel or something? This repo looks solid. I’d love to see how you built it😅

**Comment by harsh-simform** [comment]
hey, @Anan7Codes no I don't have a youtube channel as of now. but I have written blogs on dev.to platform. 
https://dev.to/hmake98
currently, I am working on writing blogs to explain more things about architecture and deployment.

**Reply by Anan7Codes** [reply]
That’s great. Nice work btw.
